applications/ContactStructuralMechanicsApplication/custom_processes/aalm_adapt_penalty_value_process.h

```diff
@@ -114,16 +114,16 @@ class AALMAdaptPenaltyValueProcess
         
         // We iterate over the node
         NodesArrayType& nodes_array = mrThisModelPart.Nodes();
-        const int num_nodes = static_cast<int>(nodes_array.size());
+        const std::size_t num_nodes = static_cast<int>(nodes_array.size());
         
         #pragma omp parallel for 
-        for(int i = 0; i < num_nodes; i++) 
+        for(std::size_t i = 0; i < num_nodes; i++) 
```

Comment:
It won't compile in Windows with VS. OpenMP 2.0 does not allow the use of size_t in the loop

Reply:
OK, when I did the change I asked @roigcarlo , then he told me that MS changed the support again

Reply:
We are still supporting the VS2015 and I'm not sure if it improves in this aspect. It would be great if you verify it.